V/VLFeat/build_tarballs.jl

```diff
@@ -0,0 +1,81 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "VLFeat"
+version = v"0.9.21"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/vlfeat/vlfeat.git", "2f6abbf13fe7ee052bc970480efaca9d6b0c195d"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd vlfeat
+
+atomic_patch -p1 ../patches/vlfeat-march.patch
+atomic_patch -p1 ../patches/vlfeat-glibc-limitation.patch
+atomic_patch -p1 ../patches/vlfeat-mingw-lowercase-windows-include.patch
+atomic_patch -p1 ../patches/vlfeat-windows.patch
+
+make_args=""
+
+if [[ $target != x86_64-* ]]; then
+    make_args+=" \
+        DISABLE_AVX=1 \
+        DISABLE_SSE2=1"
+fi
+
+if [[ $target == aarch64-apple-darwin*
+    || $target == x86_64-apple-darwin* ]]; then
+    ARCH=maci64
+elif [[ $target == i686-w64-mingw32 ]]; then
+    ARCH=win32
+elif [[ $target == x86_64-w64-mingw32 ]]; then
+    ARCH=win64
+elif [[ $target == i686-linux-*
```

Comment:
Can also check the `nbits` [environment variable](https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables)

Reply:
Modified as you suggested.